core/src/mmu/gb_mmu.rs

```diff
@@ -167,6 +171,58 @@ impl<'a, D:AudioDevice, G:GfxDevice, J:JoypadProvider> GbMmu<'a, D, G, J>{
             mem_watch: crate::debugger::MemoryWatcher::new()
         };
         if bootrom_missing{
+            if mode == Mode::CGB {
+                // Mimic the CGB bootrom behavior
+                if cgb_reg & BIT_7_MASK != 0{
+                    mmu.write(KEY0_REGISTER_ADDRESS, cgb_reg, 0);
+                }
+                else{
+                    mmu.write(KEY0_REGISTER_ADDRESS, 0x4, 0);   // Set bit 2 that indicates DMG compatability mode 
+                    mmu.write(OPRI_REGISTER_ADDRESS, 1, 0);     // Set DMG priority mode
+
+                    // Setup the default BG palletes
+                    mmu.write(BGPI_REGISTER_ADDRESS, BIT_7_MASK, 0);    // Set to auto increment
+                    // 0xFFFFFF
+                    mmu.write(BGPD_REGISTER_ADDRESS, 0xFF, 0);
+                    mmu.write(BGPD_REGISTER_ADDRESS, 0x7F, 0);
```

Comment:
Extract those to a method and add a comment with the default palettes along with explanation about the palettes